packages/side-nav/test/accessibility.test.js

```diff
@@ -0,0 +1,42 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, nextRender } from '@vaadin/testing-helpers';
+import '../enable.js';
+import '../vaadin-side-nav-item.js';
+import '../vaadin-side-nav.js';
+
+describe('accessibility', () => {
+  describe('side nav item', () => {
+    let item;
+
+    beforeEach(() => {
+      item = fixtureSync('<vaadin-side-nav-item></vaadin-side-nav-item>');
+    });
+
+    describe('ARIA roles', () => {
+      it('should set "listitem" role on side-nav-item', () => {
+        expect(item.getAttribute('role')).to.equal('listitem');
+      });
+    });
```

Comment:
This is already tested with snapshot tests, so it can be removed, I guess.

Reply:
Removed the test.